client.go

```diff
@@ -258,6 +258,7 @@ type Client struct {
 	startWriterOnce   sync.Once
 	replyWithoutQueue bool
 	unusable          bool
+	pingPongConfig    *PingPongConfig
```

Comment:
Thank you!

Let's avoid pointer here, only keep pingInterval and pongTimeout as Duration.

Also, this struct is aligned in memory to consume less RAM (as we can have many connections, trying to save as much as possible here) - so better to move pingInterval and pongTimeout as Duration after `lastPing` for example.

Reply:
@FZambia Sure thing. Do you want me to open a new pr for that?

Reply:
@FZambia I added a new commit. lmk if you want a new pr.

Reply:
No-no, just modify this one - no real reason to create new. Just add commits as you like - we are squashing anyway when merging

Reply:
Yep, thanks - added one more comment, otherwise it seems good to me

Reply:
And one more :)